public/static/locales/de/editProfile.json

```diff
@@ -28,26 +28,33 @@
     "firstName": "Vorname",
     "lastName": "Nachname",
     "name": "Name der {{type}}",
-    "email": "E-Mail",
+    "email": "Email",
     "address": "Adresse",
     "city": "Stadt",
-    "zipCode": "PLZ",
+    "zipCode": "Postleitzahl",
     "country": "Land",
-    "bio": "Profilbeschreibung",
+    "bio": "Profil Beschreibung",
```

Comment:
old one was better

Reply:
Thanks, changed it, now it shows the old one again.